packages/playwright-msw/src/utils.ts

```diff
@@ -66,8 +67,60 @@ export const convertMswPathToPlaywrightUrl = (path: Path): RegExp => {
   );
 };
 
-export const wait = (delay: number): Promise<void> => {
-  return new Promise((resolve) => {
-    setTimeout(resolve, delay);
-  });
-};
+export function objectifyHeaders(headers: Headers): Record<string, string> {
+  const result: Record<string, string> = {};
+  headers.forEach((value, key) => (result[key] = value));
+  return result;
+}
+
+function isNullArray(arr: Uint8Array): boolean {
+  return (
+    arr.length === 4 && arr.every((el, idx) => el === 'null'.charCodeAt(idx))
```

Comment:
Why the check on `4`? What is `4`? 
Maybe put it in a constant?

Reply:
Why not `arr.byteLength === 0`?

Reply:
This naive function is a way of checking if the Uint8Array is an array of char codes that represent the literal string `null`. I think this is slightly different to the behavior of `arr.byteLength === 0`.

I'm not entirely sure why this was necessary early on, but I was having issues with converting the readable stream from MSW to a buffer. If the value was "null", the route wouldn't get handled correctly. In retrospect, I suspect this was either due to me just not mocking things correctly, or just misinterpreting the issue entirely. I'm able to remove this logic and it seems to work just fine. 🤷